packages/frontend-2/lib/common/utils/downloadManager.ts

```diff
@@ -0,0 +1,23 @@
+export type ManagerExtension = 'exe' | 'dmg'
+
+// Util to download the Manager file
+export const downloadManager = (extension: ManagerExtension) => {
```

Comment:
@fabis94 Regarding 2, this snippet is used in both FE1 and FE2, I reused that. I added a promise because the browser can reject downloading files this way. We have a mixpanel even for succesful downloads and if the file itsn't actually downloaded it's misleading to throw that event

Reply:
@Mikehrn The try-catch makes sense then, but why the promise? There's no async logic here. And even if there were any, I'd think you could write it w/ async/await syntax instead of having to create a manual promise

Reply:
@fabis94 Yeah true, I'll remove that